### PR TITLE
Update ember cli

### LINKED ISSUE #246: Setup cross-browser tests
As reported in https://github.com/emberjs/ember-qunit/issues/298, after updating to the newer API's here some previously passing test suites fail on older browsers.

We should:

* Add testing for the following in Sauce Labs / Browserstack
  * IE11
  * Safari
  * Edge
* Ensure the following are ran in headless mode in CI directly:
  * Firefox
  * Chrome

### PR DESCRIPTION
**Pull request by cibernox**
Implements part of #246

### CODE REVIEW

**Comment by rwjblue** [comment]
There is one failure on Ember 2.0 only:

```
not ok 407 Firefox 56.0 - wait helper tests: it can wait only for AJAX
    ---
        actual: >
            Local Data!Remote Data!Remote Data!
        expected: >
            Local Data!Remote Data!
        stack: >
```

**Comment by Turbo87** [comment]
@rwjblue I'm not sure that failure is actually related to the PR. I think that test already failed once on `master` and restarting the test run "solved" it. We might have a flaky test here 🤔

**Comment by cibernox** [comment]
@rwjblue can you rebuild and see what happens?

**Comment by Turbo87** [comment]
restarted

**Comment by cibernox** [comment]
Green this time

**Comment by cibernox** [comment]
@rwjblue This adds firefox to CI, but I suppose you have to be authorized by saucelabs for the safari/IE/Edge part

**Comment by rwjblue** [comment]
Awesome, thank you for tackling this @cibernox!

File: .travis.yml [hunk 1]
```diff
@@ -67,4 +67,4 @@ install:
   - yarn install
 
 script:
-  - yarn ember try:one $EMBER_TRY_SCENARIO --skip-cleanup
+  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO --skip-cleanup
```
Comment (Turbo87) [inline .travis.yml]: @cibernox why?

File: package.json [hunk 2]
```diff
@@ -32,18 +32,18 @@
...
     "ember-cli-dependency-checker": "^2.0.0",
-    "ember-cli-eslint": "^4.0.0",
+    "ember-cli-eslint": "^4.2.1",
     "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-pretender": "^1.0.1",
-    "ember-cli-shims": "^1.1.0",
     "ember-cli-test-loader": "^2.2.0",
+    "ember-cli-shims": "^1.2.0",
```
Comment (Turbo87) [inline package.json]: can you move that up so that the list is alphabetical?

File: tests/helpers/start-app.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-import { run } from '@ember/runloop';
-import { merge } from '@ember/polyfills';
 import Application from '../../app';
 import config from '../../config/environment';
+import { merge } from '@ember/polyfills';
+import { run } from '@ember/runloop';
```
Comment (Turbo87) [inline tests/helpers/start-app.js]: why did you change the order here?

File: .travis.yml (hunk 1 above)
Comment (cibernox) [reply .travis.yml]: It's the autogenerated line by `ember init`

File: tests/helpers/start-app.js (hunk 3 above)
Comment (cibernox) [reply tests/helpers/start-app.js]: It's also the default output by `ember init`

File: .travis.yml (hunk 1 above)
Comment (Turbo87) [reply .travis.yml]: I know... we should change that... 😉

Comment (rwjblue) [reply .travis.yml]: haha, sounds good (when using `yarn` at least)
